2019/harness/test.js

```diff
@@ -367,6 +373,10 @@ TestExecutor.prototype.startNextTest = function() {
     }
   };
 
+  if (harnessConfig.fullscreen) {
+    this.currentTest.video.requestFullscreen();
```

Comment:
I would see this more inside of `start()`, or where the handling of the `mute` flag is located at.

Reply:
I'm not familiar with this code, but just a question - is it possible that requestFullscreen() would fail? If so, how would that be handled?

Reply:
I imagine that it can, I'll look into that.

Reply:
I've moved the `requestFullscreen` call to `getNewVideoTag` (where muted is handled). I've also added a try/catch so that when a fullscreen request fails, the test will fail.